include/swift/AST/PrintOptions.h

```diff
@@ -548,7 +548,8 @@ struct PrintOptions {
   /// attributes.
   ///
   /// \see swift::emitSwiftInterface
-  static PrintOptions printSwiftInterfaceFile(bool preferTypeRepr,
+  static PrintOptions printSwiftInterfaceFile(ModuleDecl *M,
```

Comment:
Please document the new parameter.

Reply:
I'm not sure; in this context, what does it mean for a module to be "current"? This makes sense only if the reader knows that `PrintOptions` contains a field with this name, but not everyone may know that.

In this context, the module being passed here is the module to be printed. So what do you think of either explaining this in the comment above or renaming `M` to `ModuleToPrint`?

Reply:
`ModuleToPrint` would be a good name.

Reply:
Done.